healthcare/api-client/v1/consent/README.rst

```diff
@@ -0,0 +1,170 @@
+.. This file is automatically generated. Do not edit this file directly.
+
+Cloud Healthcare API Python Samples
+===============================================================================
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=healthcare/api-client/v1/consent/README.rst
+
+
+This directory contains samples for Cloud Healthcare API. `Cloud Healthcare API`_ implements healthcare-native protocols and formats to accelerate ingestion, storage, analysis, and integration of healthcare data with cloud-based applications.
+- See the `migration guide`_ for information about migrating to Python client library v0.25.1.
+
+.. _migration guide: https://cloud.google.com/vision/docs/python-client-migration
```

Comment:
This migration guide is for the Cloud Vision client. This is referencing a `google-cloud` client from 2017. https://pypi.org/project/google-cloud/0.25.1/

Reply:
Done.